app/javascript/article-form/elements/moreConfig.jsx

```diff
@@ -1,8 +1,19 @@
 import { h, Component } from 'preact';
 import PropTypes from 'prop-types';
-import { generateMainImage } from '../actions';
 
 // const ImageManagement = ({ onExit }) => (
+
+const handleInputText = ({ label, id, value, onKeyUp }) => {
+  return (
+    <div>
+      <label htmlFor={id}>
+        {label}
+        <input type="text" value={value} name={id} onKeyUp={onKeyUp} id={id} />
```

Comment:
do we need the `id` attribute? it wasn't there before

Reply:
To avoid es-lint `eslint(jsx-a11y/label-has-associated-control)` @rhymes